### PR TITLE
Post-update logging

### PR DESCRIPTION
**Pull request by pfrenssen**
Fix for #3308 

This is postponed on #3302 and #3303.

### CODE REVIEW

**Comment by pfrenssen** [comment]
The tests show that practically no useful output is being logged during the execution of the post-updates.

This can be fixed by moving the post-update code into a batch operation inside Drush, similar to how it was done for `hook_update_N()` updates in `UpdateDBCommands::updateDoOne()`.

**Comment by claudiu-cristea** [comment]
Discussed with @pfrenssen and I will review this PR tomorrow morning so it can be quickly ready for the 9 stable release.

File: src/Commands/core/UpdateDBCommands.php [hunk 1]
```diff
@@ -210,17 +210,98 @@ public function updateDoOne($module, $number, $dependency_map, &$context)
...
+
+    /**
+     * Batch command that executes a single post-update.
+     *
+     * @param string $function
+     *   The post-update function to execute.
+     * @param array $context
+     *   The batch context.
+     */
+    public function updateDoOnePostUpdate($function, &$context)
```
Comment (claudiu-cristea) [inline src/Commands/core/UpdateDBCommands.php]: It seems that post updates have their own way of logging. Sad that we can not reuse the Drupal code.

**Comment by claudiu-cristea** [comment]
Thank you. The exception message is now logged. Review done. @weitzman, this can be committed.

**Pull request merged**